dispatch/bin/dispatch-admin

```diff
@@ -44,8 +49,52 @@ def start_project(name):
         os.path.join(os.getcwd(), name)
     )
 
+def run_tests():
+
+    os.environ['DJANGO_SETTINGS_MODULE'] = 'dispatch.tests.settings'
+
+    django.setup()
+
+    TestRunner = get_runner(settings)
+    test_runner = TestRunner()
+
+    failures = test_runner.run_tests(
+        [os.path.join(PACKAGE_DIR, 'tests')]
+    )
+
+    if failures:
+        sys.exit(1)
+
+def coverage_report():
+
+    # Setup coverage
+    cov = coverage.Coverage(
+        branch=True,
+        source=['dispatch'],
+        omit=[
+            os.path.join(PACKAGE_DIR, '*/migrations/*'),
+            os.path.join(PACKAGE_DIR, 'tests/*'),
+            os.path.join(PACKAGE_DIR, 'dispatch/bin/*')
+        ]
+    )
+
+    # Start tracking coverage
+    cov.start()
+
+    # Run tests
+    run_tests()
+
+    # Stop tracking coverage
+    cov.stop()
+    cov.save()
+
+    # Show report
+    cov.report()
```

Comment:
This currently prints out the coverage in the terminal, but if we use html_report() we can have it print out the coverage report in .html. This is a bit more overhead but it does show us which lines are missing coverage. I'll add a screenshot below of what it looks like. 
<img width="1041" alt="screen shot 2017-03-13 at 2 41 37 pm" src="https://cloud.githubusercontent.com/assets/2354516/23876547/36b34936-07fb-11e7-9e22-0a094a45c637.png">